detekt.yml

```diff
@@ -0,0 +1,65 @@
+style:
```

Comment:
Nice, @angrezichatterbox :) Feel free to make good first issues for each of these that we want to be enabled!

Reply:
I will make issues for those :).